CONTRIBUTING.md

```diff
@@ -1,12 +1,56 @@
-## Where to ask Questions?
+# How to contribute
 
-Questions can be asked in form of issues in [project_name] repository:
-[Github project issues url]
+If you would like to help us and contribute to this project, we would be happy to do so. 
```

Comment:
"If you would like to help us and contribute to this project, you are very welcome!" ?